devnets/startNetwork.ts

```diff
@@ -216,3 +214,70 @@ async function spawnNode(tempDir: string, binary: string, args: string[], signal
     }
   })
 }
+
+const authorities = [
```

Comment:
@tjjfvi I hardcoded these values until `ed25519` and `ecdsa` support is added to `capi-crypto-wrappers`

Reply:
Can these be camel cased? Otherwise, looks good